### PR TITLE
[Website] Remove short descriptions for non-blog-post layouts

### PR DESCRIPTION
**Pull request by hramos**
My earlier PR, #9648, introduced an issue that [broke the website](https://circleci.com/gh/facebook/react-native/10634). This PR addresses the issue.

### CODE REVIEW

**Comment by facebook-github-bot-2** [comment]
By analyzing the blame information on this pull request, we identified @Andreyco and @JoelMarcey to be potential reviewers.

**Comment by facebook-github-bot-1** [comment]
@hramos updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9666/files/e0f98a74a6fd94c3ba9aaa024f41d61816c95721..e033de65d7f9ecc3eb806ac6f19be3f626d59c8c)

File: website/layout/DocsLayout.js [hunk 1]
```diff
@@ -36,8 +36,7 @@ var DocsLayout = React.createClass({
     return (
       <Site
         section="docs"
-        title={metadata.title}
-        description={content.trim().split('\n')[0]} >
```
Comment (hramos) [inline website/layout/DocsLayout.js]: The issue in the earlier PR was introduced here, as content can be either undefined or JSX. On further review, using the default website description when a doc is shared should be alright, so I'm removing this.

File: website/layout/PageLayout.js [hunk 2]
```diff
@@ -28,8 +28,7 @@ var support = React.createClass({
     return (
       <Site
         section={metadata.section}
-        title={metadata.title}
-        description={content.trim().split('\n')[0]} >
```
Comment (hramos) [inline website/layout/PageLayout.js]: Similarly, we'll just default to the generic description here.

File: website/layout/BlogPageLayout.js [hunk 3]
```diff
@@ -32,7 +32,8 @@ var BlogPageLayout = React.createClass({
     return (
       <Site
         section="blog"
-        title="Blog">
+        title="React Native Blog"
+        description="The best place to stay up-to-date with the latest React Native news and events.">
```
Comment (hramos) [inline website/layout/BlogPageLayout.js]: Added a blog-specific description to the blog index layout.

File: website/layout/AutodocsLayout.js [hunk 4]
```diff
@@ -890,8 +890,7 @@ var Autodocs = React.createClass({
     return (
       <Site
         section="docs"
-        title={metadata.title}
-        description={content.trim().split('\n')[0]} >
```
Comment (hramos) [inline website/layout/AutodocsLayout.js]: This also caused issues in the earlier PR.

File: website/core/Site.js [hunk 5]
```diff
@@ -97,8 +97,8 @@ var Site = React.createClass({
         <head>
           <title>{title}</title>
           {
-            metaTags.map((tag) =>
-              <meta {...tag} />)
+            metaTags.map((tag, index) =>
+              <meta key={index} {...tag} />)
```
Comment (hramos) [inline website/core/Site.js]: This fixes [a warning](https://facebook.github.io/react/docs/multiple-components.html#dynamic-children).

**Comment by JoelMarcey** [comment]
hotfix! 

@facebook-github-bot shipit

**Comment by facebook-github-bot-6** [comment]
Thanks for importing.If you are an FB employee go to [Phabricator](https://phabricator.intern.facebook.com/D3791596) to review internal test results.

**Pull request closed without merging**